server/backend/readme.md

```diff
@@ -178,3 +178,27 @@ for test Output,
 
 - find the `output` tab
 - select `Python Test Log` in from the pulldown, top right of the output window
+
+
+## ----------------------------------------------------------------------------
+## -------------- Windows Configuration ----------------------------------------
+## ----------------------------------------------------------------------------
+
+Before you follow the steps below, ensure you have Python installed or updated to the latest version. Install and start your Docker desktop for Windows.
+
+```
+  - cd server/backend
+  - run docker compose up -d fam-flyway
+  - run this command in the same directory: python3 -m venv venv
+  - change your directory to:
+    - `cd server/backend/venv/Scripts`
```

Comment:
There is probably a way to **NOT** going into child folder (/venv/Scripts) and still can execute the activate.bat.
Have you tried that?